.github/workflows/e2e.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       - name: Checkout
         id: checkout
         uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+      - name: Update GH CLI
+        uses: dev-hanz-ops/install-gh-cli-action@v0.2.0
```

Comment:
Minor: should we pin to a specific hash? Especially as this it the private beta / interim Action.